src/components/header/admission-quick-link-tabs.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import PresetButton from '@/components/preset-button/preset-button';
+import ADMISSION_QUICK_LINK from '@/constants/admission-quick-link';
+import { AdmissionType } from '@/types/admission-type';
+import { ADDMISSION } from '@/types/admission-type';
+
+function AdmissionQuickLinkTabs({ initialAdmissionType }: { initialAdmissionType: AdmissionType | null }) {
+  const [selectedAdmissionType, setSelectedAdmissionType] = useState(initialAdmissionType ?? 'SUSI');
```

Comment:
진짜 사소한거긴한데 저는 해당 경우에 아래 방법처럼 기본값을 주는 방법을 선호하거든요...! 혹시 해당 방법에 대해서는 어떻게 생각하시나요?
```suggestion
function AdmissionQuickLinkTabs({ initialAdmissionType='SUSI' }: { initialAdmissionType: AdmissionType}) {
  const [selectedAdmissionType, setSelectedAdmissionType] = useState(initialAdmissionType);
```

Reply:
그러네요...! 너무 단순히 보이는 부분만 생각해서 말씀드린 것 같습니다! 기존 방법이 괜찮을 것 같네요!